### PR TITLE
extend download-kafka.sh script

### PR DESCRIPTION
**Pull request by mielmat**
I made changes in download-kafka.sh script according to issue description, any comments are welcome

### CODE REVIEW

File: download-kafka.sh [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+path="kafka/${KAFKA_VERSION}/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz"
+mirror=$(curl --stderr /dev/null "https://www.apache.org/dyn/closer.cgi?path=/${path}&as_json=1" | jq -r '.preferred')
 
 if [[ -z "$mirror" ]]; then
 	echo "Unable to determine mirror for downloading Kafka, the service may be down"
 	exit 1
 fi
 
-url="${mirror}kafka/${KAFKA_VERSION}/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz"
+url=${mirror}${path}
```
Comment (sscaling) [inline download-kafka.sh]: This looks a bit weird from a bash scripting perspective. Perhaps it's better to just use the response from `closer.cgi` to build the full URL, such as:

```
url=$(curl "https://www.apache.org/dyn/closer.cgi?path=${path}&as_json=1" | jq -r '"\(.preferred)\(.path_info)")'
```

Comment (mielmat) [reply download-kafka.sh]: Thanks for review, I made changes in my pull request

File: download-kafka.sh [hunk 2]
```diff
@@ -1,11 +1,12 @@
 #!/bin/sh -e
+KAFKA_VERSION=1.1.0
```
Comment (sscaling) [inline download-kafka.sh]: KAFKA_VERSION and SCALA_VERSION are provided by the Dockerfile so shouldn’t be added here.

**Comment by sscaling** [comment]
Thanks!

**Comment by sscaling** [comment]
Fixes https://github.com/wurstmeister/kafka-docker/issues/310

**Pull request merged**